### ISSUE TITLE
[features][web] multicursor

### ISSUE THREAD

**Issue by paantya** [allegroai/clearml#630]
add multicursor, like it is WandB.

I'm talking about loss charts in SCALARS.

display of reference along several axes, for the convenience of comparing values.

For example, I want to see what the value of the training loss was at a particular moment of the validation loss. I find this particular value on the validation line, and the wand is immediately duplicated for me with the value displayed on the train line, in the same step number.
You can peep at WandB for more details.

perhaps here the question arises that it is too expensive, then we can make this parameter disabled by default, and if we want to enable it, we will need to go to the panel with Smoothing and enable the corresponding flag.

as an option, make such a display only for those charts that are displayed on the screen, those that are hidden, do not do it for them. This may help with optimization.

slack thread: https://clearml.slack.com/archives/CTK20V944/p1648509660122039?thread_ts=1647900411.846749&cid=CTK20V944

**Comment by paantya** [comment]
not the best photo, only by the fact that it is impossible to make a script with the hovered cursor.

![image](https://user-images.githubusercontent.com/2975813/161131787-ef9114f3-09fc-4582-be7b-9f9727c5407a.png)

**Comment by paantya** [comment]
It can be very convenient to look like this when we compare any 2 graphs.

**Comment by ainoam** [comment]
Thanks for the suggestion @paantya.
We'll consider how this can fit into our UI roadmap and update as the roadmap solidifies.

**Comment by paantya** [comment]
Yeah thanks!
I'll wait for updates)
